scripts/main/photo.js

```diff
@@ -803,13 +803,17 @@ photo.editTags = function (photoIDs) {
 			.map((tag) => tag.trim())
 			.filter((tag) => tag !== "" && tag.indexOf(",") === -1)
 			.sort();
-		photo.setTags(photoIDs, newTags);
+		photo.setTags(photoIDs, newTags, data.override);
 	};
 
 	const setTagDialogBody = `
 		<p></p>
 		<form>
 			<div class="input-group stacked"><input class='text' name='tags' type='text' minlength='1'></div>
+			<div class='input-group compact-inverse'>
+				<label for="override">${lychee.locale["OVERRIDE"]}</label>
+				<input type='checkbox' id='override' name='override' />
+			</div>
```

Comment:
Maybe we should add an explanation here, what happens if the checkbox is not checked. In other words an helping text which says that the tags are appended to the existing tags, if the checkbox is unchecked.